catalog-rest-service/src/main/java/org/openmetadata/catalog/events/ChangeEventHandler.java

```diff
@@ -182,7 +183,7 @@ private static ChangeEvent copyChangeEvent(ChangeEvent changeEvent) {
     }
 
     var entityInterface = Entity.getEntityInterface(entity);
-    if (entityInterface.getChangeDescription() == null) {
+    if (entityInterface == null || entityInterface.getChangeDescription() == null) {
```

Comment:
Entityinterface should not be null, right? That is a bug that we should not ignore.

Reply:
You are right. Fixed it in the latest patch.